.github/workflows/project.yml

```diff
@@ -49,14 +49,15 @@ jobs:
       matrix:
         version: ['0.8', '0.10', '0.12', '4', '6', '8', '10', '12.0.0']
     runs-on: ubuntu-latest
-    env:
-      DD_INJECTION_ENABLED: 'true'
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.version }}
       - run: node ./init
+      - run: node ./init
+        env:
+          DD_INJECTION_ENABLED: 'true'
```

Comment:
`DD_INJECTION_ENABLED` is normally not a boolean but rather a comma-separated list of keywords that identify features enabled through auto-instrumentation. When SSI is in the effect, a typical value will be `DD_INJECTION_ENABLED=tracer`. When SSI is not used, the env var will simply be absent.

Reply:
Yes, but being a string means that `true` is also a valid value. The idea here was to not specify a real value and instead just provide any string.